po/pl.po

```diff
@@ -145,27 +145,27 @@ msgstr "Caps Lock jest włączony!"
 
 #: src/app/components/playback/playback_controls.rs:64
 msgid "Pause"
-msgstr ""
+msgstr "Wstrzymaj"
 
 #: src/app/components/playback/playback_controls.rs:66
 msgid "Play"
-msgstr ""
+msgstr "Odtwórz"
 
 #: src/app/components/playback/playback_controls.ui:17
 msgid "Shuffle"
 msgstr ""
 
 #: src/app/components/playback/playback_controls.ui:27
 msgid "Previous"
-msgstr ""
+msgstr "Poprzedni"
 
 #: src/app/components/playback/playback_controls.ui:50
 msgid "Next"
-msgstr ""
+msgstr "Nazsępny"
```

Comment:
A little typo here, should be "Następny" instead.

Reply:
a small typo crept in